cmd/juju/model/show.go

```diff
@@ -144,6 +144,11 @@ func (c *showModelCommand) apiModelInfoToModelInfoMap(modelInfo []params.ModelIn
 			return nil, errors.Trace(err)
 		}
 		out.ControllerName = controllerName
+		// TODO (anastasiamac 2017-06-01) Since we have changed model.Name
+		// to be a quantified model name, i.e. $owner/$name, I wonder
+		// if we should now key on model.ShortName to preserve previous
+		// behavior. We need to revisit this area anyway when re-working owner
+		// concept in the models.
```

Comment:
Looking at the changes in the tests, etc, I think we should.
Changing the top level key is going to change how people actually parse the content, and we have 'name' to give you the fully qualified name.